src/components/atoms/SkillDots/SkillDots.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC } from 'react';
```

Comment:
Może to jednak `Dots` skoro to `atom`.
`SkillDots` sugeruje, że jest do coś specyficznego i powinno być w innym miejscu.